alpha/helmfile.yaml

```diff
@@ -36,6 +36,12 @@ releases:
     version: 0.1.0
     missingFileHandler: Warn
 
+  - name: datarepo-monitoring-frontend-config   # release name
+    namespace: monitoring   # target namespace
+    chart: datarepo-helm/frontend-config   # chart name
+    version: 0.1.0
+    missingFileHandler: Warn
```

Comment:
Do we need to pass in the chart values here?

Reply:
I actually set the default to the right value so that we wouldn't have to

Reply:
(that's in the other repo)

Reply:
And the value is hardcoded in our Terraform:
https://github.com/broadinstitute/terraform-jade/blob/9f48597975045e0ce838807273e6e8127380ca4a/datarepo/modules/core-infrastructure/network.tf#L2

Reply:
Ah, that's copy pasta.  I'll remove

Reply:
k, removed from that other pr

Reply:
Oh I see! Thanks!